assets/packages/adminsidepanel/build/css/adminsidepanel.rtl.css

```diff
@@ -362,181 +384,220 @@ ol.breadcrumb.title-bar-breadcrumb {
   .ls-flex .grow-5, .ls-flex-row .grow-5, .ls-flex-column .grow-5 {
-    flex-grow: 5; }
+    -webkit-box-flex: 5;
+            flex-grow: 5; }
   .ls-flex .shrink-5, .ls-flex-row .shrink-5, .ls-flex-column .shrink-5 {
     flex-shrink: 5; }
   .ls-flex .col-5, .ls-flex-row .col-5, .ls-flex-column .col-5 {
     width: 41.66667%; }
   .ls-flex .grow-6, .ls-flex-row .grow-6, .ls-flex-column .grow-6 {
-    flex-grow: 6; }
+    -webkit-box-flex: 6;
+            flex-grow: 6; }
   .ls-flex .shrink-6, .ls-flex-row .shrink-6, .ls-flex-column .shrink-6 {
     flex-shrink: 6; }
   .ls-flex .col-6, .ls-flex-row .col-6, .ls-flex-column .col-6 {
     width: 50%; }
   .ls-flex .grow-7, .ls-flex-row .grow-7, .ls-flex-column .grow-7 {
-    flex-grow: 7; }
+    -webkit-box-flex: 7;
+            flex-grow: 7; }
   .ls-flex .shrink-7, .ls-flex-row .shrink-7, .ls-flex-column .shrink-7 {
     flex-shrink: 7; }
   .ls-flex .col-7, .ls-flex-row .col-7, .ls-flex-column .col-7 {
     width: 58.33333%; }
   .ls-flex .grow-8, .ls-flex-row .grow-8, .ls-flex-column .grow-8 {
-    flex-grow: 8; }
+    -webkit-box-flex: 8;
+            flex-grow: 8; }
   .ls-flex .shrink-8, .ls-flex-row .shrink-8, .ls-flex-column .shrink-8 {
     flex-shrink: 8; }
   .ls-flex .col-8, .ls-flex-row .col-8, .ls-flex-column .col-8 {
     width: 66.66667%; }
   .ls-flex .grow-9, .ls-flex-row .grow-9, .ls-flex-column .grow-9 {
-    flex-grow: 9; }
+    -webkit-box-flex: 9;
+            flex-grow: 9; }
   .ls-flex .shrink-9, .ls-flex-row .shrink-9, .ls-flex-column .shrink-9 {
     flex-shrink: 9; }
   .ls-flex .col-9, .ls-flex-row .col-9, .ls-flex-column .col-9 {
     width: 75%; }
   .ls-flex .grow-10, .ls-flex-row .grow-10, .ls-flex-column .grow-10 {
-    flex-grow: 10; }
+    -webkit-box-flex: 10;
+            flex-grow: 10; }
   .ls-flex .shrink-10, .ls-flex-row .shrink-10, .ls-flex-column .shrink-10 {
     flex-shrink: 10; }
   .ls-flex .col-10, .ls-flex-row .col-10, .ls-flex-column .col-10 {
     width: 83.33333%; }
   .ls-flex .grow-11, .ls-flex-row .grow-11, .ls-flex-column .grow-11 {
-    flex-grow: 11; }
+    -webkit-box-flex: 11;
+            flex-grow: 11; }
   .ls-flex .shrink-11, .ls-flex-row .shrink-11, .ls-flex-column .shrink-11 {
     flex-shrink: 11; }
   .ls-flex .col-11, .ls-flex-row .col-11, .ls-flex-column .col-11 {
     width: 91.66667%; }
   .ls-flex .grow-12, .ls-flex-row .grow-12, .ls-flex-column .grow-12 {
-    flex-grow: 12; }
+    -webkit-box-flex: 12;
+            flex-grow: 12; }
   .ls-flex .shrink-12, .ls-flex-row .shrink-12, .ls-flex-column .shrink-12 {
     flex-shrink: 12; }
   .ls-flex .col-12, .ls-flex-row .col-12, .ls-flex-column .col-12 {
     width: 100%; }
 
 .ls-flex-item {
-  flex: 1; }
+  -webkit-box-flex: 1;
+          flex: 1; }
   .ls-flex-item .grow-1 {
-    flex-grow: 1; }
+    -webkit-box-flex: 1;
+            flex-grow: 1; }
   .ls-flex-item .col-1 {
     width: 8.33333%; }
   .ls-flex-item .grow-2 {
-    flex-grow: 2; }
+    -webkit-box-flex: 2;
```

Comment:
Not related to the commit or pull request

? Why `-webkit` here ? Outdated dependance ?

https://caniuse.com/mdn-css_properties_flex-grow